src/lib/lambda.js

```diff
@@ -79,8 +80,39 @@ const getFunctionsInRegion = async region => {
 	return loop();
 };
 
+const getAllLambdasCount = async AWS => {
+	const allLambdasPromises = regions.map(region => getFunctionsInRegion(region, AWS));
+	const allStacks = await Promise.all(allLambdasPromises);
+
+	return _.flatten(allStacks).length;
+};
+
+const deleteLambda = async (lambda, AWS) => {
+	const Lambda = new AWS.Lambda({ region: lambda.region });
+	await Lambda.deleteFunction({ FunctionName: lambda.functionName }).promise();
+};
+
+const deleteAllLambdas = async AWS => {
```

Comment:
can you rename to `deleteAllFunctions` to be consistent with existing function names?